pom.xml

```diff
@@ -180,6 +180,8 @@
                 <version>${maven.compiler.plugin.version}</version>
                 <configuration>
                     <release>11</release>
+                    <compilerArgument>-g:vars</compilerArgument>
```

Comment:
What do you use these arguments for?

Reply:
The idea is to preserve variable names in method signatures so they can be used by tooling. I came across this in the context of Jackson modules. I am not 100% yet whether this is working or how to test it. Also does not hurt though. ;)

Reply:
Admittedly, with what I have learned about plugin management recently, I might be that we need to move this into the `pluginManagement` section.